query/src/sql/sql_parser.rs

```diff
@@ -558,34 +292,13 @@ impl<'a> DfParser<'a> {
         }
     }
 
-    fn parse_create_database(&mut self) -> Result<DfStatement, ParserError> {
-        let if_not_exists =
-            self.parser
-                .parse_keywords(&[Keyword::IF, Keyword::NOT, Keyword::EXISTS]);
-        let name = self.parser.parse_object_name()?;
-        let (engine, engine_options) = self.parse_database_engine()?;
-
-        let create = DfCreateDatabase {
-            if_not_exists,
-            name,
-            engine,
-            engine_options,
-            options: HashMap::new(),
-        };
-
-        Ok(DfStatement::CreateDatabase(create))
-    }
-
     fn parse_describe(&mut self) -> Result<DfStatement, ParserError> {
         if self.consume_token("stage") {
-            let obj_name = self.parser.parse_object_name()?;
-            let desc = DfDescribeStage { name: obj_name };
-            Ok(DfStatement::DescribeStage(desc))
+            self.parse_desc_stage()
+        } else if self.consume_token("table") {
+            self.parse_desc_table()
         } else {
-            self.consume_token("table");
-            let table_name = self.parser.parse_object_name()?;
-            let desc = DfDescribeTable { name: table_name };
-            Ok(DfStatement::DescribeTable(desc))
+            self.expected("stage or table", self.parser.peek_token())
```

Comment:
This will cause unit tests and stateless tests to fail. Maybe it should be deleted.